tests/test_importers_multi.py

```
@@ -112,6 +112,72 @@ def test_import_chatgpt_export_basic(tmp_path):
+    _write(export_dir / "conversations.json", conversations)
+
+    out_dir = tmp_path / "out"
+    first = import_chatgpt_export(
+        export_dir,
+        output_dir=out_dir,
+        collapse_threshold=10,
+        html=False,
+        html_theme="light",
+        force=False,
+    )
+    assert first and not first[0].skipped
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)